### DISCUSSION TITLE
How to prevent API from execute the URL before shortening It?

### DISCUSSION THREAD

**Discussion by wfbraga** [YOURLS/YOURLS#2946] [Q&A]
I have a URL that has many security parameters that are verified before redirecting the visitor to the final destination (the resulting URL).

The API is visiting the secure URL and shortening the resulting URL.

The resulting URL is useless for the visitor. Only through the secured URL, the visitor can get the final content.

**Comment by wfbraga** [comment]
I have got it.
If I use the POST method and pass the title parameter it won't visit the URL before shortening it.

**Reply by ozh** [reply]
Providing the title will bypass YOURLS fetching the remote URL to get its title, yes. Be it POST or GET.
